.github/workflows/test-build-nautobot-action.yaml

```diff
@@ -0,0 +1,85 @@
+---
+# This file contains the important snippets from `ci_integration.yml` to be tested on feature branch push.
+#     Images `push` is hardcoded to `false` here.
+# TBD: Remove this file after review and before merge.
+name: "Testing workflow"
+on:  # yamllint disable
+  push:
+    branches:
+      - "u/snaselj-4028-ci-integration-workflows"
+
+jobs:
+  container-build:
+    name: "Build Container Images (GHCR Only)"
+    runs-on: "ubuntu-22.04"
+    if: |
+      github.event_name == 'push' &&
+      (github.ref_name == 'u/snaselj-4028-ci-integration-workflows')
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+    steps:
+      - name: "Configuration"
+        id: "config"
+        shell: "bash"
+        run: |
+          # This is a testing workflow, so we hardcode the branch name
+          export BRANCH="snaselj"
+          # export BRANCH="${{ github.ref_name }}"
+          export LATEST="false"
+          export LATEST_BRANCH="false"
+          export LATEST_PY="false"
+
+          if [[ $BRANCH == "develop" ]]; then
+            export LATEST_PY="true"
+          fi
+
+          if [[ "${{ matrix.python-version }}" == "3.11" ]]; then
+            export LATEST_BRANCH="true"
+            if [[ $LATEST_PY == "true" ]]; then
+              export LATEST="true"
+            fi
+          fi
+
+          echo "branch=$BRANCH" >> $GITHUB_OUTPUT
+          echo "latest=$LATEST" >> $GITHUB_OUTPUT
+          echo "latest-branch=$LATEST_BRANCH" >> $GITHUB_OUTPUT
+          echo "latest-py=$LATEST_PY" >> $GITHUB_OUTPUT
```

Comment:
This reads unintuitive for me:

The names suggest:
`LATEST`: "Should be tagged with `latest`"
`LATEST_BRANCH`: "This is a branch that can have any `latest` tag applied"
`LATEST_PY`: "This is the latest Python version supported"

But as I read the implementation it seems `LATEST_PY` and `LATEST_BRANCH` are flipped.

Reply:
Variables has better naming now.